app/routers/event.py

```diff
@@ -112,11 +190,7 @@ def by_id(db: Session, event_id: int) -> Event:
     try:
         event = db.query(Event).filter_by(id=event_id).one()
     except NoResultFound:
-        error_message = f"Event ID does not exist. ID: {event_id}"
-        logger.exception(error_message)
-        raise HTTPException(
-            status_code=status.HTTP_404_NOT_FOUND,
-            detail=error_message)
+        nonexisting_event(event_id)
```

Comment:
Please leave the logging in tact :)

Reply:
I didn't touch it, just moved it to a separate function so that I can reuse it - is that not ok?

Reply:
Ah, I haven't noticed. kk